profiles/car.lua

```diff
@@ -584,6 +586,14 @@ function way_function (way, result)
     result.backward_speed = math.min(penalized_speed, scaled_speed)
   end
 
+  -- Handle high frequency reversible oneways (think traffic signal controlled, changing direction every 15 minutes).
+  -- Scaling speed to take average waiting time into account plus some more for start / stop.
+  if oneway and "alternating" == oneway then
+    local scaling_factor = 0.4
```

Comment:
should this check for valid speeds? If the speed is explicitly set to `math.huge`, scaling here could have unwanted side-effects. Right?